### PR TITLE
bug fixes for the tempo converter

### PR DESCRIPTION
**Pull request by jeromebarre**
## Description

In the proxy data there are a lot of inconsistencies in how the mask is specified from a file to another.
The changes allow the converter to run on proxy files and produce ioda files for our tests period.

## Dependencies

None

## Impact

None

## Checklist

- [ x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ x] I have run the unit tests before creating the PR

### CODE REVIEW

File: src/compo/tempo_nc2ioda.py [hunk 1]
```diff
@@ -121,6 +124,7 @@ def _read(self):
                 sfp = np.ma.array(sfp, mask=mask)
                 ak = ncd.groups['support_data'].variables['surface_pressure'].Eta_A
                 bk = ncd.groups['support_data'].variables['surface_pressure'].Eta_B
+                print(np.shape(mask))
```
Comment (BenjaminRuston) [inline src/compo/tempo_nc2ioda.py:127]: is this print still needed, if so a little more description would likely make it even more helpful ;0)

**Review by BenjaminRuston — APPROVED** [review]
bring on some TEMPO data

File: src/compo/tempo_nc2ioda.py (hunk 1 above)
Comment (jeromebarre) [reply src/compo/tempo_nc2ioda.py:127]: no it was just for testing, removing it now

File: src/compo/tempo_nc2ioda.py [hunk 2]
```diff
@@ -70,6 +70,7 @@ def _read(self):
         # loop through input filenames
         first = True
         for f in self.filenames:
+            print(f)
```
Comment (mer-a-o) [inline src/compo/tempo_nc2ioda.py:73]: there's one more print here.

**Review by mer-a-o — APPROVED** [review]

**Pull request merged**